### ISSUE TITLE
Newbie user error, unable to call requestVastChain

### ISSUE THREAD

**Issue by thedubb** [unruly/vast-parser#4]
This isn't an issue, just a newbie user error.  I'm having a difficult time setting up a basic test page.  I created an _index.html_ file and used the _vast-parser.amd.js_ script but i'm unable to call the _vastAdManager.requestVastChain_ to load the vast .xml that I would like to parse.  Is there a little more detail you can give about setup or maybe provide a working html file?

**Comment by mrwilson** [comment]
Hi,
Can you show us the source of the test page you made?
I've opened a new issue to create a sample page for the project.

Thanks!

**Comment by thedubb** [comment]
I ended up scrapping the index.html page, and then tried using an import...

`import {vastAdManager} from 'src/js/vastAdManager';`

...but that didn't work either, I'm not in front of the code so I don't remember what the error was exactly.  At one point in my index.html file I tried calling vast-parser.amd.js via the <script> tag because it looks like the rollup.config.js bundles all .js files into that but ended up with a defined
can't be defined which led me down the requirejs path. I didn't feel like I was getting anywhere with that so I decided to reach out in hopes to get an example of the correct implementation.  A sample HTML page would be much appreciated.
